### DISCUSSION TITLE
Zee Cinema

### DISCUSSION THREAD

**Discussion by shuaibuubah** [iptv-org/community#647] [Q&A]
**Channel Name:** Zee Cinema
**Channel ID:** ZeeCinema.in
**Country:** India
**Language:** Hindi/English
**External Link (optional):** xxx
**Notes (optional):** xxx

**Comment by BellezaEmporium** [comment]
This channel was already asked by someone else. Please look on the issues list before adding a new one here.

Reference : iptv-org/iptv#1934

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19650
